scripts/apps/authoring/multiedit/multiedit.ts

```diff
@@ -53,6 +54,7 @@ function MultieditService(storage, superdesk, authoringWorkspace: AuthoringWorks
     };
 
     this.exit = function(item) {
+        this.items.forEach((item) => sdApi.article.unlock(item.article));
```

Comment:
`unlock` operation is asynchronous. You should wait for all calls to finish before updating UI. If some calls fail, check if those articles are locked at the moment, if yes - show a message that some articles failed to unlock. Regardless do close multi edit view.